tests/integration/test_relation.py

```diff
@@ -67,6 +68,29 @@ async def build_and_deploy_ingress():
     await ops_test.model.wait_for_idle()
 
 
+@pytest_asyncio.fixture(name="anycharm_update_ingress_config")
+async def anycharm_update_ingress_config_fixture(request, ops_test, run_action):
+    """Temporarily update the ingress relation data using anycharm update_ingress method."""
+    config = copy.deepcopy(request.param)
+    await run_action(
+        ANY_APP_NAME,
+        "rpc",
+        method="update_ingress",
+        kwargs=json.dumps({"ingress_config": config}),
+    )
+    await ops_test.model.wait_for_idle(status="active")
+
+    yield config
+
+    await run_action(
+        ANY_APP_NAME,
+        "rpc",
+        method="update_ingress",
+        kwargs=json.dumps({"ingress_config": {k: "" for k in config.keys()}}),
```

Comment:
I think the `.keys()` can be omitted since the dictionary will yield keys if used in an iteration context. Do you think calling the variable `key` instead of `k` makes the code more readable?

`kwargs=json.dumps({"ingress_config": {key: "" for key in config}})`

Reply:
Yes, you are right, removed, thanks!